server/src/settings.example.cfg

```diff
@@ -37,3 +37,7 @@ GITHUB_TOKEN = REDACTED
 ISSUES_URL = https://api.github.com/repos/hackforla/311-data/issues
 PROJECT_URL = REDACTED
 GITHUB_SHA = DEVELOPMENT
+
+[Redis]
+ENABLED = True
```

Comment:
Can we default this to false?
Also is there a url for redis? Im guessing its defaulting to localhost:6379
In prod we will have a environment variable override for this

OOORRRRR we can base `ENABLED` off of 'was there a redis url provided or not' that way we kill two birds with 1 stone

Reply:
yeah I can change that. 

for the url, right now it's using 'redis' as the host, which I guess is the hostname of the redis service within the docker compose network...if I understand docker correctly. And the port is 6379. I'll figure out how to set this up as an env variable

Reply:
ohhh okay yeah...makes sense
Within the docker env itll be exposed as both redis:6379 and localhost:6379 so either way we want to be explicit with how we address redis so we can override it with heroku's `fn8934fh3o8g3o3qhg893gg.heroku.com:6379`

Reply:
ok cool this is done